metadata/base/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+resources:
+- deployment.yaml
+- service.yaml
+
```

Comment:
use `namePrefix: metadata-`

Reply:
This is not necessary since is it is under "metadata" dir.

Reply:
yes, it is not necessary, but as [best practice](https://github.com/kubeflow/manifests#Best-practices-for-kustomize-targets), we'd better follow this rule.

Reply:
To follow the rule, I will make the change.

but I think the rule means to be used for a directory containing multiple targets. Grouping files by targets using the same prefix makes sense, e.g. /katib/base. 

For a directory with a single target, the prefix adds overhead but no value. e.g. /tensorboard/base/ gets it right.